index.html

```diff
@@ -5,9 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <title>My Portfolio</title>
 </head>
 <body>
-    
+    <div class="header">
+        <a href="#"><img src="/image/images.png" width="100px"></a>
+        <nav> 
+        <a href="#" class="fa fa-bars"></a>
+        </nav>
+    </div>
+
+    <section>
+        <h1>I'm Abhishek. <br> Glad to see <br> you !</h1>
+        <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. </p>
+        <a href="https://www.linkedin.com/in/abhishek-kausik-600aa6218/">LET'S CONNECT</a>
+            <ul class="social">
+                <li><a href="https://m.facebook.com/abhisekh.kaushik.9?eav=AfaA4ohBeaoQ7IVUNJE5Nbz2VYSv_cPf7PJDGNyp-G9M9dzmuaBOBWLsNX-WrA9ha-Y&paipv=0" class="fa fa-facebook"></a></li>
+                <li><a href="https://www.instagram.com/sketcher_abhii/" class="fa fa-instagram"></a></li>
+                <li><a href="https://www.linkedin.com/in/abhishek-kausik-600aa6218/" class="fa fa-linkedin"></a></li>
+                <li><a href="https://twitter.com" class="fa fa-twitter"></a></li>
+                <li><a href="https://github.com/Nomarg/" class="fa fa-github"></a></li>
+            </ul>
+    </section>
```

Comment:
Your design
![image](https://github.com/Nomarg/My-Portfolio/assets/78251927/54e1549f-1f53-4c4c-bf38-5550e34ff1c4)

Please kindly add all the section background images (according to the design). so that your website will look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 🛠🤗

Figma
![image](https://github.com/Nomarg/My-Portfolio/assets/78251927/ae4b24d5-4d80-48d3-9958-2d96aeaecffd)